docs/source/api/index.md

```diff
@@ -235,14 +223,14 @@ which sometimes requires updating default values, for example to use new algorit
     :value: True
 
     Whether to return pointwise values when computing the
-    :data:`information criterion <stats.information_criterion>`.
+    :data:PSIS-LOO-CV.
```

Comment:
```suggestion
    PSIS-LOO-CV.
```

The :data: part was for that to render as a cross-reference to the other rcParam. Now it should be regular text